packages/material-ui-lab/src/MobileDateRangePicker/MobileDateRangePicker.tsx

```diff
@@ -272,7 +272,7 @@ const MobileDateRangePicker = makeDateRangePicker('MuiPickersDateRangePicker', M
    *  renderInput={(startProps, endProps) => (
    *   <React.Fragment>
    *     <TextField {...startProps} />
-   *     <DateRangeDelimiter> to <DateRangeDelimiter>
+   *     to
```

Comment:
Why do we use Box in the docs but no Box internally?

Reply:
The assumption is that we don't need a `DateRangeDelimiter` component, it's use case is too narrow to make it any more valuable than a Box.

Reply:
I was not asking if we need `DateRangeDelimiter`. I was asking why the internal implementation is different than the publicly documented one.

Reply:
By internal implementation, do you mean the JSDoc? I was aiming for making it as simple as possible.

Reply:
No I mean the actual code. Docs use `<Box sx={{ mx: 2 }}> to </Box>` while internal impl uses `to`

Reply:
I don't understand, there are no internal implications, everything is developer side.

Reply:
Instead of asking others to repeat what they said, you could try to understand them. Right now your priority is justifying what you did is the correct thing. Resolved in [`a14d145` (#24298)](https://github.com/mui-org/material-ui/pull/24298/commits/a14d14591bdcef6a713346993c97163c12f26667)

Reply:
Ohh, so you did meant the JSDoc :)

Reply:
The incentive was that using Box in the prop's description is subjective, it depends on the context of somebody using the date range picker. It was also so we don't have to update the JSDoc if we move from `<Box sx={{ mx: 2}}>` to `<x.div mx={2}>`. It also makes it easier to copy and paste.